src/controllers/usersController.js

```diff
@@ -122,6 +122,58 @@ export const getAll = async (req, res) => {
   }
 };
 
+export const getOne = async (req, res) => {
+  try {
+    const { userId } = req.params;
+    const user = await User.findOne({
+      where: { id: userId },
+      attributes: [
+        'id',
+        'firstName',
+        'lastName',
+        'nationalId',
+        'phone',
+        'email'
+      ]
+    });
+
+    if (!user) {
+      return errorRes(res, 404, res.__('User not found'));
+    }
+    return successRes(res, 200, res.__('Successfully got a user'), user);
+  } catch (error) {
+    return errorRes(res, 500, res.__('There was a server error'));
+  }
+};
+
+export const deleteOne = async (req, res) => {
+  try {
+    const { userId } = req.params;
+    const user = await User.findOne({
+      where: { id: userId },
+      attributes: [
+        'id',
+        'firstName',
+        'lastName',
+        'nationalId',
+        'phone',
+        'email'
```

Comment:
I think you didn't need to add attributes cause you wanted to delete

Reply:
ok let me work on it